### ISSUE TITLE
Some icons do not work

### ISSUE THREAD

**Issue by vovkkk** [andreibsk/win10-theme#3]
E.g. Python files have ![](https://raw.githubusercontent.com/andreibsk/win10-theme/master/icons/file_type_source.png) instead of ![](https://raw.githubusercontent.com/andreibsk/win10-theme/master/icons/file_type_python.png)
Markdown has ![](https://raw.githubusercontent.com/andreibsk/win10-theme/master/icons/file_type_markup.png), not ![](https://raw.githubusercontent.com/andreibsk/win10-theme/master/icons/file_type_markdown.png)

I guess there is need for tmPreferences files to assign icon to syntax, e.g. https://github.com/nfour/Sublime-Theme-Cola/tree/master/icons

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>scope</key>
    <string>text.html.markdown,text.html.markdown.multimarkdown</string>
    <key>settings</key>
    <dict>
        <key>icon</key>
        <string>file_type_markdown</string>
    </dict>
</dict>
</plist>
```

**Comment by andreibsk** [comment]
You're right. I missed that because I'm using the Insider Preview and on that build the icons somehow work by default.

**Comment by andreibsk** [comment]
How is it now with 11cb013cc0ebae9336dc2b1432f994c5a4a149c9?

**Comment by vovkkk** [comment]
Alright! thank you.

Seems like there is no definition for `file_type_markup.png` which would be `text.xml`?

Would be nice if settings icon supported other scopes like `source.sublime-settings` (and similar) from PackageDev, but they are not default, so I’d understand if you’ll refuse.
